core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -156,6 +157,8 @@ public KafkaSchemaRegistry(SchemaRegistryConfig config,
                            + " node where all register schema and config update requests are "
                            + "served.");
     this.masterNodeSensor.add(m, new Gauge());
+    this.zkAclsEnabled = config.getBoolean(SchemaRegistryConfig.ZOOKEEPER_SET_ACL_CONFIG) &&
+            JaasUtils.isZkSecurityEnabled();
```

Comment:
Wouldn't we want to throw an exception if the user configures `ZOOKEEPER_SET_ACL_CONFIG` but `isZkSecurityEnabled` is false?